content/ember-5-0-released.md

```diff
@@ -0,0 +1,330 @@
+deprecated code targeted for that major version.
+When it comes to introducing new features, Ember generally aims to ship new features in minor releases, offering backwards compatibility for existing code at the same time as giving developers the chance to try out new capabilities.
+This approach reduces the challenges that teams face for major upgrades, compared to producing big, breaking, splashy major versions with lots of new features.
+
+<!-- alex ignore retext-equality -->
+In Ember 4.x minor releases, we landed a bunch of features (along with various bug fixes, of course):
+
+- In [Ember 4.1](https://blog.emberjs.com/ember-4-1-released/), we shipped a bunch of quality-of-life improvements:
+    - the `@cached` decorator for caching expensive getters
+    - the `@service` decorator so you don't have to `import { inject as service }` anymore
+    - the `refresh` method on the Ember router, making it easier to refresh the data for the whole current route tree
+
+- In [Ember 4.4](https://blog.emberjs.com/ember-released-4-4/), we shipped a `{{unique-id}}` helper to make it easier to write accessible forms and made it easier to customize tests with improved blueprints for `setupTest`.
+
+- In [Ember 4.5](https://blog.emberjs.com/ember-4-5-released/), we made it possible to use plain old JavaScript functions as helpers: no need to wrap them with `helper()` any more!
+    <!-- alex ignore retext-equality -->
+    We also shipped a new testing primitive, `renderSettled`, which lets you wait for when Ember has re-rendered but *other* promises are still in flight—handy for testing your loading spinners, for example.
+
+- In [Ember 4.8](https://blog.emberjs.com/ember-4-8-released/), we shipped a preview of our native TypeScript support, starting out with hand-authored types.
+
+- In [Ember 4.9](https://blog.emberjs.com/ember-4-9-released/), we did a *ton* of internal polish work on our types and started testing Ember itself against our supported range of TypeScript versions.
+
+- In [Ember 4.10](https://blog.emberjs.com/ember-released-4-10/), we introduced a new, carefully-designed API for Ember's Dependency Injection system: `Owner` is a subset of the `ApplicationInstance` type which covers all of its APIs with less duplication and confusion about what to use.
+
+- In [Ember 4.11](https://blog.emberjs.com/ember-released-4-11/), we published the first TypeScript types in Ember generated from Ember's own source code.
+
+- In [Ember 4.12](https://blog.emberjs.com/ember-released-4-12/), we enabled generating TypeScript blueprints with `ember generate`.
+
+We also made significant strides in key parts of the ecosystem outside the core framework itself over the past 18 months:
+
+- We committed to shipping a new component format using `<template>` tags to get the best parts of “single-file components” and formats like JSX, while keeping the best parts of Ember’s declarative Handlebars templates.
```

Comment:
I'm highly wary of the phrase 'single file components', because it's not quite accurate. 

Svelte and Vue use single file components for sure
React and <template> 're multi-component files... Which isn't a catchy name.

Maybe this could say something like:

We committed to shipping a new component format using `<template>` tags to get the best parts of multi-language files and formats like JSX, while keeping the best parts of Ember’s declarative templates


I also removed the word handlebars, because i don't think we should use it. New learners have been lied to, unintentionally, and end up using handsebars' own docs for ember's templates, which then has them confused and wasting time with things that don't and won't ever work.

This is, in part, why i have the tutorial sites that i do. Even if partially incomplete, they can at least be correct 🙃

Reply:
```suggestion
- We committed to shipping a new component format using `<template>` tags to get the best parts of “single-file components” and formats like JSX, while keeping the best parts of Ember’s declarative templates.
```